### ISSUE TITLE
Resolve cyclic dependency of CapX project RTP number and request document

### ISSUE THREAD

**Issue by ianhinder** [UoMResearchIT/CapX#356]
It's not possible to create a CapX project (and hence RTP number) without a request document, and request documents have the RTP number in their name and at the top of the document.

The policy is that "the request document should be completed before the project is entered into CapX". Should we use XXX in the request document filename etc, even when shared with researchers?

I suspect that in practice, people are creating the projects in CapX to use them for resourcing and scheduling calculations before the request documents are finished, and putting a dummy request link in until the doc is finished. Can we improve the process so this is not necessary?

**Comment by aharwood2** [comment]
We can -- suggestions? I've marked this as documentation as I think it is possible that the outcome is an update to the PSM guide rather than the software but we'll see. Happy to discuss this.

**Comment by aharwood2** [comment]
I guess you can create the request doc with no RTP entered, save to SharePoint, and simultaneously create the Capx entry and then paste the link in. Once the Capx entry is saved then you can add the number to the request doc. Sure there is a back and forth but you just have to create the two at the same time. Once we add more statuses the project will sit in the "awaiting write-up" state anyway so it is acceptable for the request doc to be a draft.

**Comment by ianhinder** [comment]
Sounds fine.  From a process point of view, my understanding was that the request doc is written before the decision is made on whether the project is in scope or not, because that decision can only be made once we have understood the request. So the above process might lead to out-of-scope projects being entered into CapX.  We presumably wouldn't create any tasks for them before they had been deemed in-scope, but is this ok?

**Comment by aharwood2** [comment]
I think we can still stick with this process. Essentially the request doc only gets given an RTP reference when it is deemed in scope? I wonder whether we should not bother even saving the request doc centrally until it is deemed in scope to save having a lot of documents named <date>_<PI>_RTP-XXX? There _may_ be some benefit of keeping the presumably partially completed documents for projects that are eventually deemed out of scope but I'm not sure. Happy to settle here on whatever we think is practical.

**Comment by aharwood2** [comment]
Added an "out of scope" folder to the projects folder to store out of scope request docs.

**Comment by aharwood2** [comment]
I think we decided that we are OK with a temporary name for the request doc on SharePoint until we confirm demand an are able to generate the RTP number with CapX so closing.
